### PR TITLE
frontends: move frontends/android/goserver go backend/mobileserver

### PR DESCRIPTION
**Pull request by benma**
An iOS app in frontends/ios would use the same package, just with a different build target.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -85,6 +85,7 @@ clean:
 	rm -rf ${WEBROOT}/build
 	cd frontends/qt && $(MAKE) clean
 	cd frontends/android && $(MAKE) clean
+	cd frontendS/mobileserver && $(MAKE) clean
```
Comment (Beerosagos) [inline Makefile:88]: typo: frontendS

**Review by Beerosagos — CHANGES_REQUESTED** [review]
App building works, but looks like there is still some `goserver` somewhere and the mobileserver.aar is not correctly ignored by git. After building my git status is:

```
beerosagos@kiddo:~/shift/repo/beero/bitbox-wallet-app$ git status
On branch pr/2454
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   frontends/android/BitBoxApp/mobileserver/mobileserver.aar

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	frontends/android/BitBoxApp/goserver/
	frontends/android/goserver/

no changes added to commit (use "git add" and/or "git commit -a")
```

Also, I'm wondering if it would make sense to move mobileserver outside the frontends folder, one level up (if that's possible). No strong opinion though

File: Makefile (hunk 1 above)
Comment (benma) [reply Makefile]: fixed

**Comment by benma** [comment]
> App building works, but looks like there is still some `goserver` somewhere and the mobileserver.aar is not correctly ignored by git. After building my git status is:
> 
> ```
> beerosagos@kiddo:~/shift/repo/beero/bitbox-wallet-app$ git status
> On branch pr/2454
> Changes not staged for commit:
>   (use "git add <file>..." to update what will be committed)
>   (use "git restore <file>..." to discard changes in working directory)
> 	modified:   frontends/android/BitBoxApp/mobileserver/mobileserver.aar
> 
> Untracked files:
>   (use "git add <file>..." to include in what will be committed)
> 	frontends/android/BitBoxApp/goserver/
> 	frontends/android/goserver/
> 
> no changes added to commit (use "git add" and/or "git commit -a")
> ```
> 

It is actually in .gitignore, but I also committed it by accident, so it was tracked. I removed it from the commit now.

> Also, I'm wondering if it would make sense to move mobileserver outside the frontends folder, one level up (if that's possible). No strong opinion though

Can do, or maybe to backend/mobileserver (bridgecommon is also in backend/bridgecommon)? Let me know what you like best, I am okay with any location.

**Comment by Beerosagos** [comment]
> Can do, or maybe to backend/mobileserver (bridgecommon is also in backend/bridgecommon)? Let me know what you like best, I am okay with any location.

backend/mobileserver looks good! :+1:

**Comment by benma** [comment]
> > Can do, or maybe to backend/mobileserver (bridgecommon is also in backend/bridgecommon)? Let me know what you like best, I am okay with any location.
> 
> backend/mobileserver looks good! 👍

Moved there!

**Review by Beerosagos — APPROVED** [review]
tACK!

**Pull request merged**